Dockerfile

```diff
@@ -0,0 +1,85 @@
+FROM	ubuntu:20.04	as	mist-build
+
+ENV	DEBIAN_FRONTEND=noninteractive
+
+WORKDIR	/src
+
+RUN	apt update -yq \
+	&& apt install -yqq build-essential cmake git
+
+RUN	git clone https://github.com/cisco/libsrtp.git \
+	&& mkdir -p libsrtp/build \
+	&& cd libsrtp/build \
+	&& cmake -DCMAKE_INSTALL_PREFIX=/src/compiled .. \
+	&& make -j$(nproc) install \
+	&& cd /src \
+	&& git clone -b dtls_srtp_support --depth=1 https://github.com/livepeer/mbedtls.git \
+	&& cd mbedtls \
+	&& mkdir build \
+	&& cd build \
+	&& cmake -DCMAKE_INSTALL_PREFIX=/src/compiled .. \
+	&& make -j$(nproc) install \
+	&& cd /src \
+	&& git clone https://github.com/Haivision/srt.git \
+	&& mkdir -p srt/build \
+	&& cd srt/build \
+	&& cmake .. -DCMAKE_INSTALL_PREFIX=/src/compiled -D USE_ENCLIB=mbedtls -D ENABLE_SHARED=false \
+	&& make -j$(nproc) install
+
+ENV	LD_LIBRARY_PATH="/src/compiled/lib" \
+	C_INCLUDE_PATH="/src/compiled/include"
+
+COPY	.	.
+
+RUN	mkdir -p build/ \
+	&& cd build/ \
+	&& cmake .. -DPERPETUAL=1 -DLOAD_BALANCE=1 -DCMAKE_INSTALL_PREFIX=/opt -DCMAKE_PREFIX_PATH=/src/compiled -DCMAKE_BUILD_TYPE=RelWithDebInfo \
```

Comment:
Let's add the flag to make shared binaries with `libmist.a` here.

(And getting each flag on its own line wouldn't be a terrible idea)

Reply:
added this in recent updates to dockerfile